lib/src/features/authenticate/presentation/ui/widgets/login_widget.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:dome_smart_home_app/src/features/authenticate/presentation/bloc/authentication_event.dart';
+import 'package:dome_smart_home_app/src/features/authenticate/presentation/ui/widgets/input_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../bloc/authentication_bloc.dart';
+
+class LoginWidget extends StatelessWidget {
+  const LoginWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints.expand(),
+      decoration: const BoxDecoration(
+        image: DecorationImage(
+            image: AssetImage("assets/images/login_background.png"),
+            fit: BoxFit.cover),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(48.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          mainAxisSize: MainAxisSize.max,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const Spacer(
+              flex: 3,
+            ),
+            const Text(
+              'Log in',
+              style: TextStyle(fontSize: 40, fontWeight: FontWeight.bold),
+            ),
+            const SizedBox(height: 30),
+            Form(
```

Comment:
what do you need this form for?

Reply:
Because they should be. Not used now, but doesn't hurt to have this